### PR TITLE
v0.16.0 release

### PR DESCRIPTION
**Pull request by rg911**
## [0.16.0] - 09-Dec-2019

**Milestone**: Fushicho.3

- Added epochAdjustment introduced in catapult-server v0.9.1.1 (Fushicho.3)
- Added `setMaxFee` to `Transaction` which calculate max transaction fee using `feeMultiplier * transaction.size`.
- Optimised `resolveAlias` implementation in transaction using object spread operator.
- Fixed couple of bugs in `blockHttp` and unit tests
- Fixed travis-CI issue
- Fixed `coveralls` always return 100% issue

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "stripInternal": true,
     "outDir": "dist",
     "allowJs": false,
+    "skipLibCheck": true,
```
Comment (dgarcia360) [inline tsconfig.json]: What's the rationale for this change?

Comment (rg911) [reply tsconfig.json]: It looks like travis was always trying to install the latest typescript 3.7.3 where we are only on 2.7.2 for now.
